robot/impl/robot_framesystem_test.go

```diff
@@ -30,7 +30,7 @@ import (
 func TestFrameSystemConfigWithRemote(t *testing.T) {
 	logger := logging.NewTestLogger(t)
 	// make the remote robots
-	remoteConfig, err := config.Read(context.Background(), rutils.ResolveFile("robot/impl/data/fake.json"), logger.Sublogger("remote"))
+	remoteConfig, err := config.Read(context.Background(), rutils.ResolveFile("robot/impl/data/fake.json"), logger.RootSublogger("remote"))
```

Comment:
I was only running tests in impl to suss out build errors. I suspect there are other _test files that are failing CI and need a similar change.